cli/template/base/next.config.mjs

```diff
@@ -20,3 +22,14 @@ const config = {
 };
 
 export default config;
+
+function checkNodeVersion() {
+  const nodeVersion = process.versions.node;
+  const [major, minor] = nodeVersion.split(".").map((n) => parseInt(n));
+  if (!major || !minor) {
+    throw new Error(`Unable to parse Node version: ${nodeVersion}`);
+  }
+  if (major < 18 || (major === 18 && minor < 6) || (major === 19 && minor < 7)) {
```

Comment:
should this be `minor < 16`?

Reply:
16 is EOL

Reply:
I think you misread `minor`, his check is "<18.6" not "<18.16"

Reply:
yes, thanks :)